sktime/datasets/tests/test_data_io.py

```diff
@@ -1080,3 +1081,41 @@ def test_write_dataframe_to_ts_fail(tmp_path):
             path=str(tmp_path),
             problem_name="GunPoint",
         )
+
+
+def test_convert_tsf_to_dataframe():
```

Comment:
The name of the test to `test_load_tsf_to_dataframe`

Reply:
Modified the function name.